ghcide/test/exe/FuzzySearch.hs

```diff
@@ -65,6 +65,7 @@ replaceAt t i c =
 dictionaryPath :: FilePath
 dictionaryPath = "/usr/share/dict/words"
 
+{-# ANN dictionary ("HLint: ignore Avoid restricted function" :: String) #-}
```

Comment:
Is this still needed? I think I would prefer this exception to be added to the `.hlint.yaml` file, but this is just my opinion.

Reply:
I think it is better to keep this annotation. It is warning about the use of `unsafePerformIO` which should be considered per function in my opinion.